recipe/meta.yaml

```diff
@@ -38,12 +38,16 @@ requirements:
 # Skipped pytest by dependencies packages missing reason. 
 # pyro-api >=0.1.1 and funsor >=0.4.1
 test:
+  source_files:
+    - test
   imports:
     - numpyro
   commands:
     - pip check
+    - pytest -v --ignore=test/test_pickle.py test/test_handlers.py
```

Comment:
In this way you are telling `pytest` to run the tests that are only in the file `test/test_handlers.py`. Is this the purpose?